app/mailers/notification_mailer.rb

```diff
@@ -18,7 +18,7 @@ def refill_medication(reminder)
   end
 
   def notification_email(recipientid, data)
-    @data = JSON.parse(data)
+    @data = HashWithIndifferentAccess.new(JSON.parse(data))
     @recipient = User.where(id: recipientid).first
     subject = 'if me | '
```

Comment:
Sure, but the way I thought once all the notifications get extracted they will have the full subject.